terraform/aws/variables.tf

```diff
@@ -35,5 +35,5 @@ variable "name" {
 }
 
 variable "image" {
-  default = "bionic-18.04"
+  default = "ami-0e6273fe5a9a1ad93"
 }
```

Comment:
Does it work for regions that differ from eu-west-2 ?

Reply:
Very good question! ami's are region-specific, we need to define a different ami for each group of validators/public nodes (same as we need to define a different region for each of them).